adm_program/modules/groups-roles/groups_roles_function.php

```diff
@@ -290,7 +290,7 @@
 
         $filename = FileSystemUtils::getSanitizedPathEntry($filename);
 
-        header('Content-Type: text/x-vcard; charset=iso-8859-1');
```

Comment:
As I understand https://de.wikipedia.org/wiki/VCard the content-type text/x-vcard should be still used for vcard version 3.0